test/test_model.py

```diff
@@ -767,6 +768,40 @@ def test_storage_tool_errors(self):
                 run()
             self.assertEqual(fake_script_calls(self, clear=True), calls)
 
+    def test_network_get(self):
+        network_get_out = ('{"bind-addresses":[{"mac-address":"","interface-name":"",'
+                           '"addresses":[{"hostname":"","value":"192.0.2.2","cidr":""}]}]'
+                           ',"egress-subnets":["192.0.2.2/32"],"ingress-addresses":["192.0.2.2"]}')
+        fake_script(self, 'network-get', f'''[ "$1" = deadbeef ] && echo '{network_get_out}' || exit 1''')
+        network_info = self.backend.network_get('deadbeef')
+        self.assertEqual(network_info, json.loads(network_get_out))
+        self.assertEqual(fake_script_calls(self, clear=True), [['network-get', 'deadbeef', '--format=json']])
+
+        network_info = self.backend.network_get('deadbeef', 1)
+        self.assertEqual(network_info, json.loads(network_get_out))
+        self.assertEqual(fake_script_calls(self, clear=True), [['network-get', 'deadbeef', '-r', '1', '--format=json']])
+
+    def test_network_get_errors(self):
+        err_no_endpoint = "ERROR no network config found for binding \"$2\""
+        err_no_rel = "ERROR invalid value \"$3\" for option -r: relation not found"
```

Comment:
I wonder about this (and similar) lines. As soon as I need to backslash a double quote, in Python, I'd change the type of the outer quotes... that is, in this case, I think I'd write this like
```suggestion
        err_no_rel = 'ERROR invalid value "$3" for option -r: relation not found'
```

above you use triple-single-quotes and then inside use double and single, so it doesn't seem to be stylistic.

Anyway, just wondering about it.